skrub/_minhash_encoder.py

```diff
@@ -93,26 +95,24 @@ class MinHashEncoder(TransformerMixin, BaseEstimator):
 
     Examples
     --------
+    >>> import pandas as pd
     >>> from skrub import MinHashEncoder
     >>> enc = MinHashEncoder(n_components=5)
 
     Let's encode the following non-normalized data:
 
-    >>> X = [['paris, FR'], ['Paris'], ['London, UK'], ['London']]
+    >>> X = pd.Series(['paris, FR', 'Paris', 'London, UK', 'London'], name='city')
     >>> enc.fit(X)
     MinHashEncoder(n_components=5)
 
     The encoded data with 5 components are:
 
     >>> enc.transform(X)
-    array([[-1.78337518e+09, -1.58827021e+09, -1.66359234e+09,
-            -1.81988679e+09, -1.96259387e+09],
-           [-8.48046971e+08, -1.76657887e+09, -1.55891205e+09,
-            -1.48574446e+09, -1.68729890e+09],
-           [-1.97582893e+09, -2.09500033e+09, -1.59652117e+09,
-            -1.81759383e+09, -2.09569333e+09],
-           [-1.97582893e+09, -2.09500033e+09, -1.53072052e+09,
-            -1.45918266e+09, -1.58098831e+09]])
+             city_0        city_1        city_2        city_3        city_4
+    0 -1.783375e+09 -1.588270e+09 -1.663592e+09 -1.819887e+09 -1.962594e+09
+    1 -8.480470e+08 -1.766579e+09 -1.558912e+09 -1.485745e+09 -1.687299e+09
+    2 -1.975829e+09 -2.095000e+09 -1.596521e+09 -1.817594e+09 -2.095693e+09
+    3 -1.975829e+09 -2.095000e+09 -1.530721e+09 -1.459183e+09 -1.580988e+09
```

Comment:
We should certainly switch to scipy-doctest at some points: https://github.com/scipy/scipy_doctest

Reply:
yep, looks super useful!